libs/platform/src/lib/components/form/form-options.ts

```diff
@@ -7,3 +7,23 @@ export type FormZone = 'zTop' | 'zBottom' | 'zLeft' | 'zRight';
 export type Column = 1 | 2 | 3 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
 
 export type Status = 'success' | 'error' | 'warning' | 'default' | 'information';
+
+export interface ColumnLayout {
+    XL?: number;
+    L?: number;
+    M?: number;
+    S?: number;
+}
+
+export interface InlineLayout {
+    XL?: boolean;
+    L?: boolean;
+    M?: boolean;
+    S?: boolean;
+}
+
+export enum RESPONSIVE_BREAKPOINTS {
```

Comment:
Can't we have this in more generic place?

Reply:
will check with kevin.

Reply:
Also it is worth keeping it configurable providing it through the injection mechanism, in this case an user can override these constants

Reply:
It is injectable now.

Reply:
values related to form only is kept in `form-option file` and values related to inline-layout has been moved to `inline-layout-collection-based.input file`. which makes more sense.
review comment has been addressed, so marking it resolved.